src/Rings/mpoly-ideals.jl

```diff
@@ -1325,3 +1326,47 @@ function minimal_generating_set(I::MPolyIdeal{<:MPolyElem_dec}; ordering::Monomi
 
   return elem_type(R)[ R(f) for f in gensS ]
 end
+
+################################################################################
+#
+# Grassman Pluecker Ideal
+#
+################################################################################
+
+@doc Markdown.doc"""
+    grassman_pluecker_ideal(subspace_dimension::Int, ambient_dimension::Int, ring::Union{MPolyRing, Nothing} = nothing)
+
+Given an ambient dimension and a subspace dimenson dimension return the ideal
+generated by the plucker relations. If the ring is not specified return the ideal in the multi variate polynomial ring over the rationals.
+
+# Examples
+```jldoctest
+julia> R, x = PolynomialRing(ResidueRing(ZZ, 7), "x" => (1:2, 1:3), ordering=:degrevlex)
+(Multivariate Polynomial Ring in 6 variables x[1, 1], x[2, 1], x[1, 2], x[2, 2], ..., x[2, 3] over Integers modulo 7, nmod_mpoly[x[1, 1] x[1, 2] x[1, 3]; x[2, 1] x[2, 2] x[2, 3]])
+
+julia> grassman_pluecker_ideal(2, 4; ring = R)
+ideal(x[1, 2]*x[2, 2] + 6*x[2, 1]*x[1, 3] + x[1, 1]*x[2, 3])
+```
+"""
+function grassman_pluecker_ideal(subspace_dimension::Int,
+                                 ambient_dimension::Int;
+                                 ring::R = nothing) where R <: Union{MPolyRing, Nothing}
```

Comment:
Hmm, don't we usually have the ring as an optional first argument? So roughly like this (untested)
```suggestion
function grassman_pluecker_ideal(subspace_dimension::Int,
                                 ambient_dimension::Int)
  return convert(Polymake.ideal.pluecker_ideal(subspace_dimension, ambient_dimension))
end

function grassman_pluecker_ideal(ring::MPolyRing,
                                 subspace_dimension::Int,
                                 ambient_dimension::Int)
  return convert(Polymake.ideal.pluecker_ideal(subspace_dimension, ambient_dimension), parent = ring)
end
```